civet.dev/reference.md

```diff
@@ -1937,6 +1937,31 @@ min := for min item of array
 max := for max item of array
 </Playground>
 
+### Object Comprehensions
+
+Loops can also accumulate their body values into an object.
+When any loop is found is found within an object expression,
```

Comment:
You should mention *braced* object expressions (or literals) here. This won't work with implicit (unbraced) object literals.